pandas/core/indexes/period.py

```diff
@@ -183,9 +183,8 @@ def __new__(
 
         if not set(fields).issubset(valid_field_set):
             raise TypeError(
-                "__new__() got an unexpected keyword argument {}".format(
-                    list(set(fields) - valid_field_set)[0]
-                )
+                "__new__() got an unexpected keyword argument "
+                f"{list(set(fields) - valid_field_set)[0]}"
```

Comment:
could do this outside f-space.

Reply:
separated that line with the new variable.